packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.js

```diff
@@ -176,26 +222,27 @@ const ErrorFormTooltip = ({
         onShown(false);
       }
     };
-
     window.addEventListener("keydown", handleTab);
     return () => {
       window.removeEventListener("keydown", handleTab);
     };
-  }, [onShown, isHelp]);
+  }, [onShown, isHelp, shown, tooltipRef, update]);
 
   return (
+    /* $FlowFixMe: https://github.com/popperjs/react-popper/issues/318 */
     <StyledFormFeedbackTooltip
-      ref={tooltipRef}
+      id={id}
+      ref={setTooltipRef}
       inputSize={inputSize}
-      position={preferredPosition}
-      shown={shown && dimensions.set}
+      shown={shown}
       isHelp={isHelp}
       data-test={dataTest}
-      inlineLabel={inlineLabel}
       aria-live="polite"
-      id={id}
-      {...dimensions}
+      placement={attrs.popper && attrs.popper["data-popper-placement"]}
+      {...popper}
     >
+      {/* $FlowFixMe: https://github.com/popperjs/react-popper/issues/318 */}
+      <StyledArrow isHelp={isHelp} ref={setArrowRef} {...arrow} />
```

Comment:
Same here. You might also be able to use some lodash function for this if you don't want to be repetitive.

Reply:
lodash is not the part of the orbit, it should be used only in orbit.kiwi.